### PR TITLE
2017-05-26 LWIMIQ

### PR DESCRIPTION
**Pull request by lgalis**
2017-05-26 LWIMIQ

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @lgalis 
@imtayadeway @syncrou @jntullo can you review this week's blog entry.

File: site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ - 59 PRs in the main [manageiq][manageiq PRs merged] repo
+ - 47 PRs in [manageiq-ui-classic][manageiq-ui-classic PRs merged] repo
+ - 18 PRs in [manaiq-ui-service][manageiq-ui-service PRs merged] repo
+ - the rest in the providers' repos: [manageiq-providers-amazon][manageiq-providers-amazon PRs merged], [manageiq-providers-azure][manageiq-providers-azure PRs merged], [manageiq-providers-vmware][manageiq-providers-vmware PRs merged]
+
+## Improved
+ 
+ - After linking the 'miq_servers' and 'miq_tasks' tables [here](https://github.com/ManageIQ/manageiq/pull/14701), to make filtering of Tasks more intuitive after merging the Tasks and Jobs screens, [PR 14679](https://github.com/ManageIQ/manageiq/pull/14679) is replacing the 'Waiting_to_start' value for the state attribute in miq_tasks table with 'Queued'.
+ - [PR 15141](https://github.com/ManageIQ/manageiq/pull/15141) Adds a new helper method for unique index columns to inventory collection
+ - [PR 15120](https://github.com/ManageIQ/manageiq/pull/15120) Use the new values in extra_vars to create a new dialog when editingan Ansible playbook service template, instead of waiting for the updated job template which may take some time.
```
Comment (jntullo) [inline site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md]: editing an?

Comment (lgalis) [inline site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md]: Thanks for noticing! Fixed.

File: site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- [PR 1354](https://github.com/ManageIQ/manageiq-ui-classic/pull/1354) - List only playbooks in the current region when creating or updating a Playbook Catalog Item
+- [PR 15124](https://github.com/ManageIQ/manageiq/pull/15124) makes TokenManager#token_ttl callable (evaluated at call time), thus delaying evaluation until call time, which honors updated settings values.
+- [PR 15112](https://github.com/ManageIQ/manageiq/pull/15112) Ensures order is qualified by table name for rss feeds
+- [PR 15151](https://github.com/ManageIQ/manageiq/pull/15151) Fixes the REST API to allow users with admin role access to requests  
+
+
+## New
+
+- [PR 15123](https://github.com/ManageIQ/manageiq/pull/15123) Allows disabling worker heartbeats for workers that are running in isolation or where communicating via DRB is not possible. This is preliminary work for 
+towards evetually allowing the spin up of workers without rake evm:start being done and activating a drb server to handle heartbeating.
```
Comment (syncrou) [inline site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md]: /evetually/eventually/

**Comment by JPrause** [comment]
Nice @lgalis ,....jam packed with PRs.
LGTM!

File: site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md (hunk 2 above)
Comment (lgalis) [inline site/_posts/2017-05-26-gearing-up-for-Gaprindashvili-and-beyond.md]: fixed - thanks @syncrou!

**Pull request merged**